<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>stepper</class>
 <widget class="QWidget" name="stepper">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>306</width>
    <height>150</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>stepper</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QWidget" name="verticalWidget" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximumSize">
      <size>
       <width>300</width>
       <height>16777215</height>
      </size>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QWidget" name="btnWidget" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QPushButton" name="stopButton">
           <property name="text">
            <string>stop</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="stepButton">
           <property name="text">
            <string>step</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="runButton">
           <property name="text">
            <string>run</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="horizontalWidget" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>speed:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSlider" name="speedSlider">
           <property name="minimum">
            <number>10</number>
           </property>
           <property name="maximum">
            <number>200</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
